src/WobblySpinner.props.js

```diff
@@ -0,0 +1,21 @@
+import { PropTypes } from "react";
+
+export const propTypes = {
```

Comment:
Understood. Reason to change is the best reason to split it out. I just see these are similar to a typing like you would in typescript, c#, java, where these would be defined w/in the class. No big deal, just pointing out.